tooling/miniutil/src/fmt/ty.rs

```diff
@@ -121,32 +120,58 @@ pub(super) fn fmt_comptypes(mut comptypes: Vec<CompType>) -> String {
 }
 
 fn fmt_comptype(i: CompTypeIndex, t: CompType, comptypes: &mut Vec<CompType>) -> String {
-    let (keyword, fields, opt_chunks, size, align) = match t.0 {
-        Type::Tuple { fields, size, align } => ("tuple", fields, None, size, align),
+    let (keyword, size, align) = match t.0 {
+        Type::Tuple { fields, size, align } => ("tuple", size, align),
```

Comment:
CI shows an "unused variable" warning here

Reply:
saw it and understood: warnings = errors with regards to PRs